source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -1354,8 +1368,14 @@ def fill_journeys(self, request_type, api_request, instance):
         )
 
         # we store the origin/destination detail in g to be able to use them after the marshall
-        g.origin_detail = origin_detail
-        g.destination_detail = destination_detail
+        import copy
+
+        g.origin_detail = copy.deepcopy(origin_detail)
+        if isinstance(g.origin_detail, dict) and 'within_zones' in g.origin_detail:
+            del g.origin_detail["within_zones"]
```

Comment:
do we have to remove it from the object? there is a serializer for within_zone?

Reply:
navitia should not expose this field in journeys response